public/locales/en/translation.json

```diff
@@ -0,0 +1,264 @@
+{
+  "c": {
+    "network": "network",
+    "favourites": "Favourites",
+    "about": "About",
+    "mynames": "My Names",
+    "learnmore": "Learn More",
+    "search": "Search",
+    "permanentregistrar": "Permanent Registrar",
+    "Registrant": "Registrant",
```

Comment:
Why are some names all lower case (eg: `search`) and others Camelcase?  Can you keep it consistent?

Reply:
It's because of what I used for labels previously and are dynamically used to get the text. It will require refactoring of some other keys inside the component.